application/src/main/java/org/thingsboard/server/controller/EntityRelationController.java

```diff
@@ -291,7 +291,7 @@ public EntityRelation getRelation(@RequestParam(FROM_ID) String strFromId,
         checkNotNull(query.getFilters());
         checkEntityId(query.getParameters().getEntityId(), Operation.READ);
         try {
-            return checkNotNull(relationService.findInfoByQuery(getTenantId(), query).get());
+            return checkNotNull(relationService.findInfoByQuery(getTenantId(), query, query.getParameters().isFetchLastLevelOnly()).get());
```

Comment:
This is wrong. Why do you pass one parameter separately, if you already pass "query" as a second parameter.